.tekton/determine-image-tag-task-oci-ta.yaml

```diff
@@ -7,12 +7,13 @@ metadata:
 spec:
   description: Determines the tag for the output image using the StackRox convention from 'make tag' output.
   params:
-  - name: tag-suffix
+  - name: TAG_SUFFIX
     description: Suffix to append to generated image tag.
     type: string
   - name: SOURCE_ARTIFACT
     description: The Trusted Artifact URI pointing to the artifact with
-      the application source code.
+      the application source code. This should be the result of the git-clone task,
+      results from other tasks might fail as dirty.
     type: string
   results:
   - name: image-tag
```

Comment:
This should also go `SCREAMING_CASE` too, for consistency: `image-tag` -> `IMAGE_TAG`.

Reply:
done in https://github.com/stackrox/stackrox/pull/11616/commits/4fc31dc77fe56c2f87d4eacb264c93c22a94c9a1